### PR TITLE
Node10UpgradeDownloadFileshareGitNugetGitRelease

### PR DESCRIPTION
**Pull request by v-nmursapall**
**Task name**: DownloadFileshareArtifacts, DownloadGithubNugetPackage,DownloadGithubReleaseV0

**Description**: Changes made as part of Node10 upgrade

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [ yes] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [yes ] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DownloadFileshareArtifactsV0/task.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 198,
+        "Minor": 200,
```
Comment (harshiniks) [inline Tasks/DownloadFileshareArtifactsV0/task.json:12]: why not 199?

File: Tasks/DownloadGitHubNugetPackageV1/task.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "demands": [],
     "version": {
         "Major": 1,
-        "Minor": 198,
+        "Minor": 200,
```
[the remark this answers is not in the document]
Comment (v-nmursapall) [reply Tasks/DownloadGitHubNugetPackageV1/task.json:20]: The current sprint is 200 , hence 200. (It is as per instructions in https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md)

File: Tasks/DownloadGitHubNugetPackageV1/task.loc.json [hunk 3]
```diff
@@ -17,7 +17,7 @@
   "demands": [],
   "version": {
     "Major": 1,
-    "Minor": 198,
+    "Minor": 200,
```
[the remark this answers is not in the document]
Comment (v-nmursapall) [reply Tasks/DownloadGitHubNugetPackageV1/task.loc.json:20]: The current sprint is 200 , hence 200. (It is as per instructions in https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md)

**Review by harshiniks — APPROVED** [review]

**Pull request merged**